examples/tests/youtube.test.ts

```diff
@@ -0,0 +1,26 @@
+// youtube test
+
+import { shortest } from "@antiwork/shortest";
+
+// Basic search and video playback
+shortest("Search for 'cute puppies' on YouTube and play the first video");
+
+// Video with specific requirements
+shortest("Watch a YouTube video and change playback settings", {
+  videoUrl: "https://www.youtube.com/watch?v=dQw4w9WgXcQ",
+  settings: {
+    quality: "1080p",
+    playbackSpeed: "1.5x",
+    subtitles: "on"
+  }
+});
```

Comment:
Show casing AI capabilities to navigate and changed settings based on the payload or configurations that is provided to it.